api/v1alpha1/aux_functions.go

```diff
@@ -0,0 +1,25 @@
+package v1alpha1
+
+func IsClientSecurityEnabled(c *EtcdCluster) bool {
+	clientSecurityEnabled := false
+	if c.Spec.Security != nil && c.Spec.Security.TLS.ClientSecret != "" {
+		clientSecurityEnabled = true
+	}
+	return clientSecurityEnabled
+}
```

Comment:
To make functions smaller - yes. Will do it.
I made them public just because they are needed in other package.
I placed them to API because seems that these functions will be api-version-independent. In other api-version we can have the same functions with different body.

I will think how to make it better. Methods or just remove from here and declare vars.

I do not like declare vars so much, because it is also code duplication and if version of api is changed, then vars will be declared differently. And this way provides more stable "interface" of api to other package.

Reply:
Done